instance/config.py

```diff
@@ -6,17 +6,20 @@ class Config(object):
     DEBUG = False
     CSRF_ENABLED = True
     SECRET = os.getenv('SECRET')
+    Database_Url = os.getenv("Main_Database")
```

Comment:
Is this really necessary, seeing that the different configurations already define their Database_url